### PR TITLE
Terraform 0.13 upgrade

### PR DESCRIPTION
**Pull request by yupwei68**
Fix https://github.com/Azure/terraform-azurerm-network/issues/44

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,47 @@ This Terraform module deploys a Virtual Network in Azure with a subnet or a set
...
+}
+```
+
+## Usage in Terraform 0.12
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
```
Comment (njuCZ) [inline README.md:44]: there is no breaking change for terraform 0.12 and 0.13 to use this module, but don't forget to add version constraints if there is a breaking change in future

File: Gemfile [hunk 2]
```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org/' do
 
 group :test do
   git 'https://github.com/Azure/terramodtest.git' do
-    gem 'terramodtest', tag: '0.5.0'
+    gem 'terramodtest', tag: '0.6.0'
```
Comment (njuCZ) [inline Gemfile:5]: may I ask what's the meaning here? it's the tag of "terramodtest" repo? from here https://github.com/Azure/terramodtest/tags I could only find the tag "0.4.0"

Comment (yupwei68) [reply Gemfile:5]: Right. It doesn't need to be released in repo. You can refer it directly in ruby.

Comment (yupwei68) [reply Gemfile:5]: It's for terraform fmt check.

**Review by njuCZ — APPROVED** [review]

**Comment by njuCZ** [comment]
LGTM

**Comment by njuCZ** [comment]
Additionally, we should update .travis.yml file to use 0.13 terraform

**Comment by yupwei68** [comment]
Hi @njuCZ Thanks for your comments.

**Comment by yupwei68** [comment]
@njuCZ Please approve again. Ruby script runs fail before and fixed now.

**Review by njuCZ — APPROVED** [review]

**Pull request merged**